### PR TITLE
3-Python

### PR DESCRIPTION
**Pull request by dahachm**
@Cakemanator

**3-Python**

9 tasks to practise python skills.

All the comments and specifications about the task in `task_*` directories.

### CODE REVIEW

File: 3-Python/task_2/task_2.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+print(f'Input: {test_3} (type: {type(test_3)})')
+printEven(test_3)
+
+print('\n    --- Case #4 ---')
+print(f'Input: {test_4} (type: {type(test_4)})')
+printEven(test_4)
+
+print('\n    --- Case #5 ---')
+print(f'Input: {test_5} (type: {type(test_5)})')
+printEven(test_5)
```
Comment (Cakemanator) [inline 3-Python/task_2/task_2.py]: What if I want to provide my own set of numbers?

File: 3-Python/task_3/task_3.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+while True:
+
+    secret = input('Tell me your secret (word/phrase): ')
+
+    if secret in dictionary:
+        print('Nice! Your secret matches.')
+        exit()
+    elif secret in dictionary:
+        print('Nice! Your secret matches.')
+        exit()
```
Comment (Cakemanator) [inline 3-Python/task_3/task_3.py]: Why do you repeat exact same condition?

File: 3-Python/task_5/task_5.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+print('  --- TEST #4 --- ')
+print('Input: ' + str(test_4))
+top3Max(test_4)
+
+print('  --- TEST #5 --- ')
+print('Input: ' + str(test_5))
+top, extras = top3Max(test_5)
+print (top)
+print (extras)
```
Comment (Cakemanator) [inline 3-Python/task_5/task_5.py]: What if I want to provide my own set of numbers?

File: 3-Python/task_2/task_2.py (hunk 1 above)
Comment (dahachm) [reply 3-Python/task_2/task_2.py]: Hi, @Cakemanator!

Thank you for feedback!

I added an option to input user's list

File: 3-Python/task_3/task_3.py (hunk 2 above)
Comment (dahachm) [reply 3-Python/task_3/task_3.py]: Looks like a result of copy-paste process, can't remember what was the point, tbh :D 

I removed this repeated condition

File: 3-Python/task_5/task_5.py (hunk 3 above)
Comment (dahachm) [reply 3-Python/task_5/task_5.py]: Also added an option for user to input his list!

**Pull request merged**